src/labkey/Pipeline.ts

```diff
@@ -48,13 +49,13 @@ export interface IGetFileStatusOptions {
      * - name: name of the file, a String.
      * - status: status of the file, a String
      */
-    success?: () => any
+    success?: RequestSuccess
```

Comment:
I assume there is a reason that this success can't just be removed and come from RequestCallbackOptions.

Reply:
Thanks for having me take a second look at this. Turns out not even `RequestSuccess` is correct in a couple of these cases. A couple of the success callbacks have "custom" function signatures which would not be properly annotated by `RequestSuccess`. Due to this I had to not use `RequestCallbackOptions` at all in these scenarios.

See https://github.com/LabKey/labkey-api-js/pull/53/commits/b7ad4d954b92df165aed88005367860acfdaed78.